utils/src/main/java/org/jboss/windup/util/ClassNameUtil.java

```diff
@@ -12,4 +13,24 @@ public static final String getPackageName(String qualifiedName)
             packageName = qualifiedName.substring(0, qualifiedName.lastIndexOf("."));
         return packageName;
     }
+    
+    /*
+     * This method assumes constants are upper cased
+     */
+    public static boolean isConstant(String fqcn)
+    {
+        return isUpperCase(fqcn);
```

Comment:
no, as Apache StringUtils does not evaluate properly non letters like undescore or dashes, so it would not help for MAX_CONSTANT only for MAXCONSTANT.